docker-compose.yml

```diff
@@ -7,10 +7,35 @@ services:
     restart: always
     ports:
       - "8888:3000"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.basic.frontend.rule=Host:explorer.raiden.network"
+      - "traefik.basic.port=8888"
+      - "traefik.basic.protocol=http"
 
   backend:
     image: backend
     build: ./backend
     restart: always
     ports:
```

Comment:
You can remove the `ports` config from the services if it's not necessary for the containers to be accessed from the host directly - traefik will communicate with them through the docker internal network.